install/playground/metallb.yaml

```
@@ -0,0 +1,13 @@
+#Replace the `addresses` field with the CIDR of Elastic IP allocated
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
+data:
+  config: |
+    address-pools:
+    - name: my-ip-space
+      protocol: layer2
+      addresses:
+      - 147.28.141.9/32 
```

```suggestion
      - 147.28.141.9/32 
      autoAssign: false
```

Only assign IPs to annotated services.